frontend/nuxt.config.ts

```diff
@@ -190,11 +132,6 @@ export default defineNuxtConfig({
     vueI18n: "./src/vue-i18n",
   },
   plausible: {
-    enabled: !isTest,
-    logIgnoredEvents: !isProductionBuild,
-    trackLocalhost: !isProdNotPlaywright,
-    autoPageviews: isProdNotPlaywright,
-    domain: import.meta.env.SITE_DOMAIN,
-    apiHost: import.meta.env.PLAUSIBLE_SITE_URL,
+    trackLocalhost: true,
```

Comment:
This setting isn't mentioned in the module documentation: https://nuxt.com/modules/plausible

How does it work?

Reply:
@sarayourfriend it's deprecated but still works, and the replacement functionality ignoredHostnames has a bug: https://github.com/nuxt-modules/plausible/issues/30

@obulat let's add a comment referencing the bug.

Reply:
I opened an upstream issue: https://github.com/nuxt-modules/plausible/issues/30
I don't know why, but `options.ignoredHostnames` always contains `localhost`, and it's not possible to remove it. So, I used the _deprecated_ setting that still works.

Reply:
Gotcha. A comment would be great, yes!